openeducat_timetable/models/timetable.py

```diff
@@ -143,7 +143,7 @@ def create(self, values):
         subtype_id = self.env['mail.message.subtype'].sudo().search([
             ('name', '=', 'Discussions')])
         if partner_ids and subtype_id:
-            for partner in partner_ids:
+            for partner in list(set(partner_ids)):
```

Comment:
In some cases it gets duplicated stuff in partner_ids then for loop below tries to create duplicated entries in mail.followers and fails because of sql_constraint.